### PR TITLE
Refactor frontend creators + docker extension

### PR DESCRIPTION
**Pull request by melnik0v**

### CODE REVIEW

File: lib/datacrafts_io_skeleton/version.rbs [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+module DatacraftsIoSkeleton
+  VERSION: String
```
Comment (overchind) [inline lib/datacrafts_io_skeleton/version.rbs]: If we want to add `rbs`, I suggest creating a separate PR for it. This file looks redundant here.

File: lib/datacrafts_io_skeleton/version.rb [hunk 2]
```diff
@@ -3,5 +3,5 @@
 module DatacraftsIoSkeleton
   # Defines gem version
 
-  VERSION = "0.2.0"
+  VERSION = "0.2.1"
```
Comment (overchind) [inline lib/datacrafts_io_skeleton/version.rb]: We didn't track any functionality changes as part of the `module structure` update. Should we do it now? Will it help us?
The current state of the skeleton seems very good to me. I suggest considering the gem release with version `0.2.0` (with this PR merged) and version upgrades with further improvements.

File: lib/datacrafts_io_skeleton.rb [hunk 3]
```diff
@@ -20,12 +20,18 @@ class Composer < ::Thor
...
-    desc "create APP_NAME", "Create you own best Rails/JS app"
+    desc "create APP_NAME", <<~DESC
+      Create your own best Rails/JS app.
+      If you have some issues with creation please use VERBOSE=true
+    DESC
 
     def create(app_name)
-      say "Skeleton in creating your best Rails/JS application \n"
+      say <<~DESC
+        Skeleton in creating your best Rails/JS application.
```
Comment (overchind) [inline lib/datacrafts_io_skeleton.rb]: It should be "Skeleton `is` creating your best Rails/JS application.", shouldn't it?

File: lib/datacrafts_io_skeleton.rb [hunk 4]
```diff
@@ -20,12 +20,18 @@ class Composer < ::Thor
...
+        If you have some issues with creation please use VERBOSE=true
```
Comment (overchind) [inline lib/datacrafts_io_skeleton.rb:31]: Should we try to add checks to avoid some type of common errors?

**Review by overchind — CHANGES_REQUESTED** [review]

File: lib/datacrafts_io_skeleton/version.rbs (hunk 1 above)
Comment (melnik0v-me) [reply lib/datacrafts_io_skeleton/version.rbs]: ohh, I tried to add rbs but remove it after several tries, so it's just a garbage now, will remove

File: lib/datacrafts_io_skeleton/version.rb (hunk 2 above)
Comment (melnik0v-me) [reply lib/datacrafts_io_skeleton/version.rb]: probably you're right

File: lib/datacrafts_io_skeleton.rb (hunk 4 above)
Comment (melnik0v-me) [reply lib/datacrafts_io_skeleton.rb:31]: currently it's tested manually and I've added this comment for situations when you have issues with your tools
sometimes local ruby or node versions or some libs has conflicts with our tool so not sure that we can test it
anyway sure we'll write specs for skeleton, but I was thinking in next release

**Review by overchind — APPROVED** [review]

**Pull request merged**